straxen/plugins/nveto_recorder.py

```diff
@@ -12,6 +12,8 @@
 @strax.takes_config(
     strax.Option('coincidence_level_recorder_nv', type=int, default=4,
                  help="Required coincidence level."),
+    strax.Option('pre_trigger_time_nv', type=int, default=150,
+                 help="Pretrigger time before coincidence window."),
```

Comment:
perhaps nice to add the unit, I assume ns but this is only a factor of 2 different from the number of samples.